css-display-3/display-contents-text-only-001.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Display: Only text on a display: contents subtree</title>
+<link rel="author" title="Emilio Cobos Álvarez" href="mailto:ecobos@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-display-3/#valdef-display-contents">
+<link rel="match" href="display-contents-text-only-001-ref.html">
+<style>
+  html, body { display: contents; }
+</style>
+PASS
```

Comment:
Are you really allowed to have display:contents on html (or other root elements)? We should have a test for that.